src/app/events/[slug]/components/ScheduleSection/ScheduleSection.tsx

```diff
@@ -10,7 +10,7 @@ type ScheduleSectionProps = {
 
 export function ScheduleSection({ schedule }: ScheduleSectionProps) {
   return (
-    <PageSection kicker="Join Us" title={schedule.title || 'Schedule'}>
+    <PageSection kicker={schedule.kicker} title={schedule.title}>
       <Tabs schedule={schedule} />
```

Comment:
Something is wrong here from before (nothing to do with your implementation) - We are passing the entire `schedule` object - Need a bit more time to look into it - Will open a new ticket for it!